backend/inference.py

```diff
@@ -33,12 +33,11 @@ def process_inference(
             task_id,
             client_id):
 
-        # Parse the JSON string
-        data = json.loads(inference.to_string())
+        # Convert the string to a list of integers
+        inference_list = [int(x) for x in inference.to_string().strip('[]').split(',')]
```

Comment:
We are sending the data with:
`fiware_node.post_data(content['data'])`
This sends only the data array, not the entire JSON object. Therefore, we are not passing the JSON as a string, but rather the list directly:
`[0, 0, 0, 0, 0, 75.28959028358858, 152.9872263988676, ...]`
As a result, there's no need to split a string here, and the list of numeric values is handled correctly without causing an error with `int(x)`.

Reply:
This format only works for the fiware_node. For real time prediction and batch prediction it does not works since the data is passed completely. Please adjust the code so it works for all use cases.